website/source/docs/agent/options.html.markdown

```diff
@@ -658,7 +658,7 @@ Consul will not enable TLS for the HTTP API unless the `https` port has been ass
 * <a name="retry_join"></a><a href="#retry_join">`retry_join`</a> Equivalent to the
   [`-retry-join` command-line flag](#_retry_join). Takes a list
   of addresses to attempt joining every [`retry_interval`](#_retry_interval) until at least one
-  [`-join`](#_join) works.
+  [`-join`](#_join) works. The list should contain IPv4 addresses without port number or bracketed IPv6 addresses with the Serf LAN port number also specified — for example: `[::1]:8301`.
```

Comment:
I think IPV4 support will work with ports as well, it should be optional for either.